Tweaks evidences (#1057)
RSE-Sheffield/RSE-Sheffield.github.io: pages/collaboration/evidence.md

---

> The RSE team at Sheffield contributed to multiple areas of the research eco-system along the years. This can take a form of research publication, software releases, committee membership, event organisations, etc. 

Reviewer: ```suggestion
The RSE team at Sheffield contribute to multiple areas of the research eco-system. Contributions can take the form of research publications, software releases, committee membership, event organisation, etc. 
```

> We are listing here all the contributions made be our team and we grouped them in five categories:

Reviewer: ```suggestion
Here we list all the contributions made by our team, grouped into five categories:
```
